backend/src/test/java/org/eclipse/tractusx/semantics/registry/AssetAdministrationShellApiSecurityTest.java

```diff
@@ -1,6 +1,5 @@
-/********************************************************************************
- * Copyright (c) 2021-2023 Robert Bosch Manufacturing Solutions GmbH
- * Copyright (c) 2021-2023 Contributors to the Eclipse Foundation
```

Comment:
removed Copyright (c) 2021-2023 Contributors to the Eclipse Foundation?

Reply:
Thanks, yes, they are the "and others" as detailed here: https://www.eclipse.org/projects/handbook/#ip-copyright-headers